browser/src/unocommands.js

```diff
@@ -251,6 +252,7 @@ var unoCommandsArray = {
 	'IncrementLevel':{text:{menu:_('Promote Outline Level'),},},
 	'IncrementSubLevels':{text:{menu:_('Promote Outline Level with Subpoints'),},},
 	'IndexesMenu':{text:{menu:_('Table of Contents and Inde~x'),},},
+	'InsertAVMedia':{global:{context:_('Insert Audio or Video'),menu:_('Media'),},},
```

Comment:
this change should be removed like its counterpart.

Reply:
Fair.
The script ought to run again, in a different PR.

Reply:
we can fix that in a different PR. 

I wonder why this is out of order....